[sdl2-mixer] Build failure with arm64-android triplet

**Host Environment**
- OS: Windows 10
- Compiler: clang from Android NDK 24.0

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install sdl2-mixer:arm64-android`

**Failure logs**
```
Computing installation plan...
The following packages will be built and installed:
    sdl2-mixer[core,nativemidi]:arm64-android -> 2.6.1#1
Detecting compiler hash for triplet arm64-android...
Restored 0 package(s) from C:\Users\Michael\AppData\Local\vcpkg\archives in 298.8 us. Use --debug to see more details.
Installing 1/1 sdl2-mixer:arm64-android...
Building sdl2-mixer[core,nativemidi]:arm64-android...
warning: -- Using community triplet arm64-android. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: C:\Projekte\Tools\vcpkg\triplets\community\arm64-android.cmake
-- Using cached libsdl-org-SDL_mixer-1cf51b6e16552e4118d90799ef752f7644efb9fd.tar.gz.
-- Cleaning sources at C:/Projekte/Tools/vcpkg/buildtrees/sdl2-mixer/src/7644efb9fd-fcdf14bbc7.clean. Use --editable to skip cleaning for the 
packages you specify.
-- Extracting source C:/Projekte/Tools/vcpkg/downloads/libsdl-org-SDL_mixer-1cf51b6e16552e4118d90799ef752f7644efb9fd.tar.gz
-- Applying patch fix-pkg-prefix.patch
-- Applying patch fix-pkgconfig.patch
-- Using source at C:/Projekte/Tools/vcpkg/buildtrees/sdl2-mixer/src/7644efb9fd-fcdf14bbc7.clean
-- Configuring arm64-android
-- Building arm64-android-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:129 (message):
    Command failed: C:/Projekte/Tools/vcpkg/downloads/tools/cmake-3.24.0-windows/cmake-3.24.0-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j5
    Working Directory: C:/Projekte/Tools/vcpkg/buildtrees/sdl2-mixer/arm64-android-dbg
    See logs for more information:
      C:\Projekte\Tools\vcpkg\buildtrees\sdl2-mixer\install-arm64-android-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/sdl2-mixer/portfile.cmake:28 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)
```

**Additional context**
From checking the attached install log file, I guess that building the sample projects isn't supported on Android. I see a `SDL2MIXER_SAMPLES` option, but I'm unsure how to pass that option in my `vcpkg.json` (manifest mode) or in my CMakeLists.txt where I do the `find_package(sdl2-mixer REQUIRED)`, or in the `CMakePresets.json file`, or in a custom triplet file.

[install-arm64-android-dbg-out.log](https://github.com/microsoft/vcpkg/files/9482732/install-arm64-android-dbg-out.log)

--- a/ports/sdl2-mixer/portfile.cmake
+++ b/ports/sdl2-mixer/portfile.cmake
@@ -23,6 +23,7 @@ vcpkg_cmake_configure(
     OPTIONS
         ${FEATURE_OPTIONS}
         -DSDL2MIXER_VENDORED=OFF
+        -DSDL2MIXER_SAMPLES=OFF
 )
 
 vcpkg_cmake_install()

--- a/ports/sdl2-mixer/vcpkg.json
+++ b/ports/sdl2-mixer/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "sdl2-mixer",
   "version": "2.6.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Multi-channel audio mixer library for SDL.",
   "homepage": "https://github.com/libsdl-org/SDL_mixer",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6698,7 +6698,7 @@
     },
     "sdl2-mixer": {
       "baseline": "2.6.1",
-      "port-version": 1
+      "port-version": 2
     },
     "sdl2-net": {
       "baseline": "2.0.1",

--- a/versions/s-/sdl2-mixer.json
+++ b/versions/s-/sdl2-mixer.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6994c429d80fd95894a448a6bb55260563cc3946",
+      "version": "2.6.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "c06711b92dfd4c048f6a56c0236697012094a6e5",
       "version": "2.6.1",
       "port-version": 1

